doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1533,10 +1533,8 @@ Computations
    | delta {? @delta_flag }
    ref_or_pattern_occ ::= @smart_qualid {? at @occs_nums }
    | @one_term {? at @occs_nums }
-   occs_nums ::= {+ @num_or_var }
-   | - @num_or_var {* @int_or_var }
-   num_or_var ::= @num
-   | @ident
+   occs_nums ::= {+ {| @num | @ident } }
+   | - {| @num | @ident } {* @int_or_var }
    int_or_var ::= @int
    | @ident
```

Comment:
It's a bit surprising that you choose to inline the definition of `num_or_var` but not `int_or_var`.

Reply:
num_or_var was only used twice, both in occs_nums.  int_or_var is used 34 times, seemed less obvious.  BTW, at some point we may want to reconsider some inlinings if they save us from describing the same details over and over rather than having a hyperlink.

Easy to change these, of course.